torch/fx/experimental/proxy_tensor.py

```diff
@@ -444,7 +444,7 @@ def getattr(self, attr, attr_val, parameter_proxy_cache):
         return attr_val
 
     def create_arg(self, a: Any):
-        if isinstance(a, torch.nn.Parameter):
+        if isinstance(a, (torch.nn.Parameter, torch._subclasses.FakeTensor)):
```

Comment:
The changes here look questionable

Reply:
In particular, it doesn't make sense to have something that matches Parameter and FakeTensor, but not regular Tensor!

Reply:
Somehow weights of fakified model show up here as `torch._subclasses.FakeTensor` 🤔 . Normally they are `nn.Parameter`.

Reply:
> In particular, it doesn't make sense to have something that matches Parameter and FakeTensor, but not regular Tensor!

For this particular instance, it seems a faked `torch.nn.Parameter` is implemented as a `torch._subclasses.FakeTensor`. There is no `FakeParameter` or similar, which makes me think this change is ok. Makes sense?